Appc 935 remove dependencies from other sdk modules

brunovarelaAptoide opened the pull request:
- Removed Retrofit and RXJava Removed.
- Appcoins_net module created
- AppcoinsClient Created
- AppcoinsHTTPClient Creted with 2 operations: PING , GET
- Log Interceptor adapted
- Wallet Utils Adapted.
- Change: only if there is a campaign the proof is sent.

brunovarelaAptoide committed:
Removed Dependencies

-Remove dependencies from appcoins core and appcoins-proxy
-Added 1 new class
- Changed build properties to test the application.

brunovarelaAptoide committed:
Created new Module Java

-Creates new module Java

brunovarelaAptoide committed:
Added Factory

-Created a factory to build AppcoinsClientFactory

brunovarelaAptoide committed:
Code Style

- Applied codestyle

brunovarelaAptoide committed:
Changed AppcoinsClientFactory

resolved an error

brunovarelaAptoide committed:
Changed AppcoinsHTTPClient

- Added class for QUeryParams

brunovarelaAptoide committed:
Created Factory and Query Params

- Factory Created
- Query Params
- MapperParams

brunovarelaAptoide committed:
Operation Class

-Create Operation abstraction

brunovarelaAptoide committed:
Mapper Class

-Defined methods for mapping params

mikieribeiro committed:
APPC-935

-HttpCliente connection done

brunovarelaAptoide committed:
Merged

-Merged HTTPCLient with Java part

brunovarelaAptoide committed:
HTTPClient Working

HTTP Client Working

mikieribeiro committed:
APPC-935

-Runnable moved from Toolbox/app to PoaManager (handleCampaing method)

mikieribeiro committed:
APPC-935

-Retrofit identification key added (//TODO REMOVER RETROFIT)

mikieribeiro committed:
APPC-935 (Retrofit Removal)

-All retrofit dependecies removed
-Any code using retrofit was commented

brunovarelaAptoide committed:
Changes on AppcoinsHTTPClient

brunovarelaAptoide committed:
Merge branch 'APPC-935-Remove-dependencies-from-other-SDK-modules' of https://github.com/Aptoide/bds-sdk into APPC-935-Remove-dependencies-from-other-SDK-modules

mikieribeiro committed:
APPC-935

-Updated Http client
-BadResponse being handled

Nuno-Monteiro committed:
Created AppcoinsClientResponse class to encapsulate a response

brunovarelaAptoide committed:
GetCampaing

la

brunovarelaAptoide committed:
Merge branch 'APPC-935-Remove-dependencies-from-other-SDK-modules' of https://github.com/Aptoide/bds-sdk into APPC-935-Remove-dependencies-from-other-SDK-modules

mikieribeiro committed:
APPC-935

-Treated http client connections exceptions
-Created LogInterceptor class

brunovarelaAptoide committed:
AppcoinsHTTPClient

la

brunovarelaAptoide committed:
Merge branch 'APPC-935-Remove-dependencies-from-other-SDK-modules' into Retrofit-dependencies-removal

brunovarelaAptoide committed:
Merge pull request #14 from Aptoide/Retrofit-dependencies-removal

Retrofit dependencies removal

brunovarelaAptoide committed:
Retrofit and RX Removed

-Retrofit and RX Removed

brunovarelaAptoide committed:
Merge branch 'APPC-935-Remove-dependencies-from-other-SDK-modules' of https://github.com/Aptoide/bds-sdk into APPC-935-Remove-dependencies-from-other-SDK-modules

brunovarelaAptoide committed:
RetroFIt and RX removed

-Retrofit and RX removed and begin creating a Interceptor

mikieribeiro committed:
APPC-935

-Log Interceptor updated
-Log Creator created

mikieribeiro committed:
Appc-395

-LogInterceptor updated

brunovarelaAptoide committed:
Changes

-Ping
-Merged Log Interceptor
- Changed to only send proof if tit has campaign

mikieribeiro committed:
APPC-935

-Internet permissions added to AndroidManifest.xml

brunovarelaAptoide committed:
Changes

-Changed Campagnemapper

brunovarelaAptoide committed:
Changes

-Removed Unused files

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/ip/IpApi.java:
```diff
@@ -1,32 +1,28 @@
 
 public interface IpApi {
-
+/*
```
remove commented code

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/ip/IpResponse.java:
```diff
@@ -1,19 +1,29 @@
 
-@JsonInclude(JsonInclude.Include.NON_NULL) public class IpResponse {
+public class IpResponse {
+  /*
```
remove commented code

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/campaign/CampaignMapper.java:
```diff
@@ -0,0 +1,77 @@
+      return val;
+    }
+
+    //TODO throw error
```
do the todo

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -56,14 +41,14 @@
   /** integer used to identify the network to wich we are connected */
   private int network = 0;
-  private CompositeDisposable compositeDisposable;
+  //private CompositeDisposable compositeDisposable;
```
remove commented code

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -206,7 +170,7 @@ public void finishProcess() {
     appPreferences.remove(PREFERENCE_WALLET_PCKG_NAME);
     poaConnector.disconnectFromService(appContext);
-    compositeDisposable.clear();
+    //compositeDisposable.clear();
```
remove commented code

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -299,29 +268,18 @@ private void checkPreferencesForPackage() {
 
     if (!processing) {
-      this.compositeDisposable = new CompositeDisposable();
+      //this.compositeDisposable = new CompositeDisposable();
```
remove commented code

Trinkes commented on appcoins-contract-proxy/src/main/java/com/asf/appcoins/sdk/contractproxy/AppCoinsAddressProxyBuilder.java:
```diff
@@ -2,9 +2,9 @@
-import retrofit2.Retrofit;
-import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
-import retrofit2.converter.jackson.JacksonConverterFactory;
+//import retrofit2.Retrofit;
```
remove commented code

Trinkes commented on appcoins-contract-proxy/src/main/java/com/asf/appcoins/sdk/contractproxy/AppCoinsAddressProxyBuilder.java:
```diff
@@ -25,11 +25,14 @@ public AppCoinsAddressProxySdk createAddressProxySdk() {
         break;
     }
-    return new Retrofit.Builder().baseUrl(baseHost)
+    //TODO REMOVER RETROFIT
```
do the todo

Trinkes commented on appcoins-contract-proxy/src/main/java/com/asf/appcoins/sdk/contractproxy/repository/RemoteRepository.java:
```diff
@@ -1,8 +1,8 @@
 import io.reactivex.Single;
-import retrofit2.http.GET;
-import retrofit2.http.Query;
+//import retrofit2.http.GET;
```
remove commented code

Trinkes commented on appcoins-contract-proxy/src/main/java/com/asf/appcoins/sdk/contractproxy/repository/RemoteRepository.java:
```diff
@@ -12,9 +12,10 @@ public RemoteRepository(ApiProvider apiProvider) {
-    return apiProvider.getApi(chainId)
-        .getAddress(contract.value)
-        .map(ProxyResponse::getWallet);
+    return null;
```
never return null on Rxjava

Trinkes commented on appcoins-contract-proxy/src/main/java/com/asf/appcoins/sdk/contractproxy/repository/RemoteRepository.java:
```diff
@@ -27,7 +28,8 @@ public RemoteRepository(ApiProvider apiProvider) {
 
   public interface Api {
-    @GET("appc/proxyaddress") Single<ProxyResponse> getAddress(@Query("contract") String contract);
+    //TODO REMOVER RETROFIT
```
do the todo

Trinkes commented on gradle.properties:
```diff
@@ -9,4 +9,24 @@ ROPSTEN_NETWORK_PROXY_CONTRACT_ADDRESS=0x3CA30A86d04e65E6E388922deCe3eBD0F100F5d
+APPCOINS_IAB_BIND_ACTION=com.appcoins.wallet.iab.action.BIND
+APPCOINS_IAB_BIND_PACKAGE=com.appcoins.wallet
+APPCOINS_IAB_PERMISSION=com.appcoins.BILLING
+APPCOINS_IAB_KEY=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAw+Qen4gtCi1triCxdb2svOsZ7OKJu19cjVDO2MNuvSw69Q+RQGYUYXQLfoFM6Nzk7tlbhIKjN2YC7/ZoxDFTzoHh1WNbLIn1X+D5p33fAUgd6vzrJLG0eLQ4A5MmXCAnaalCpgeA3BoRfmUr+ED09ycRXEhS/IU12bTN0pO3trm28SOGAkwBCsGYPBBlPvpRiydhvrcIbpfB3oEMrjvlpXo+9k8mSO0qk03gnrGp5N6inQe6k8qDQ+sdFQYPsdiDo//EGyzLwoLBEzg31DSObDJprz7rgHR2z1lDPkHf7NALD8/a49cS21SPX3IPY+YX9NI8HEALh/eiRjhQmq ...
```
remove keys from version control

Trinkes commented on gradle/versions.gradle:
```diff
@@ -41,6 +41,7 @@ ver.spongycastle_prov = "1.58.0.0"
 ver.jackson = "2.9.5"
 
 // https://mvnrepository.com/artifact/com.squareup.retrofit2/retrofit
+//TODO REMOVER RETROFIT
```
do the todo

Trinkes commented on toolbox/build.gradle:
```diff
@@ -1,42 +1,126 @@
-    targetSdkVersion TARGET_SDK_VERSION
+  defaultConfig {
+    applicationId "com.appcoins.trivialdrivesample"
+    minSdkVersion 26
```
use the project's values(min sdk 21)

Trinkes commented on toolbox/src/main/AndroidManifest.xml:
```diff
@@ -1,8 +1,9 @@
-
+  <uses-permission android:name="android.permission.INTERNET" />
   <application
+      android:usesCleartextTraffic="true"
```
why would you want this?

Trinkes commented on toolbox/src/main/java/com/asf/appcoins/toolbox/App.java:
```diff
@@ -2,16 +2,25 @@
 
 public class App extends Application {
 
+  private final String developerAddress = BuildConfig.IAB_WALLET_ADDR;
```
not used

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsClient.java:
```diff
@@ -0,0 +1,49 @@
+  private final String packageName;
+  private final int versionCode;
+  private final Interceptor interceptor;
+  private String serviceUrl;
```
why isn't this final?

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsHTTPClient.java:
```diff
@@ -0,0 +1,119 @@
+    }
+  }
+
+  @Override public String Get() {
```
use a meaningful name on methods

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsHTTPClient.java:
```diff
@@ -0,0 +1,119 @@
+    this.operation = operation;
+  }
+
+  @Override public void run() {
```
why don't you implement this in two different classes?

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsHTTPClient.java:
```diff
@@ -0,0 +1,119 @@
+    return response;
+  }
+
+  @Override public boolean Ping() {
```
methods should not start with capital letter
give a better name to this method, when you call ping, you don't expect to receive a boolean

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsHTTPClient.java:
```diff
@@ -0,0 +1,119 @@
+      }
+    } catch (MalformedURLException e) {
+      e.printStackTrace();
+    } catch (IOException e) {
```
if there is an error, how do you handle it if your're only logging it here?

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsConnectionQuerys.java:
```diff
@@ -0,0 +1,8 @@
+
+import java.io.IOException;
+
+interface AppcoinsConnectionQuerys {
```
why do you need this interface?

Trinkes commented on appcoins_net/src/main/java/com/appcoins/net/AppcoinsHTTPClient.java:
```diff
@@ -0,0 +1,119 @@
+    URL url = null;
+    HttpURLConnection urlConnection = null;
+    try {
+      url = new URL("http://www.google.com");
```
should we ping our servers instead?

brunovarelaAptoide committed:
Pull Request Changes

-Removed unused classes
-Removed TODOS
- Removed commented code
- Removed Unused code
-Change the ping service to make a request to our services.
- Rollback the classes changes from appcoins-contract-proxy
- removed key from gradle.properties
- changed min sdk to 21
- Created a AppCoinsPingClient
- Removed AppCoinsConnectionQuerys interface

brunovarelaAptoide committed:
Changes

Added a guard to null objects in AppCoinsHTTPClient

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/WalletUtils.java:
```diff
@@ -0,0 +1,57 @@
+import com.asf.appcoins.sdk.ads.BuildConfig;
+import com.asf.appcoins.sdk.ads.R;
+
+public class WalletUtils {
```
there is other class with the same name that's not used, delete it

Trinkes commented on microraidenj-bds/build.gradle:
```diff
@@ -36,7 +36,7 @@ dependencies {
   testImplementation lib.mockito_core
   androidTestImplementation lib.android_test_runner
-
+  //TODO REMOVER RETROFIT
```
todo

Trinkes commented on microraidenj-bds/src/main/java/com/bds/microraidenj/ws/BDSMicroRaidenApi.java:
```diff
@@ -31,7 +31,7 @@ static BDSMicroRaidenApi create(boolean debug) {
 
     OkHttpClient client = builder.build();
-
+    //TODO REMOVER RETROFIT
```
todo

Trinkes commented on toolbox/build.gradle:
```diff
@@ -1,42 +1,126 @@
+    debug {
       minifyEnabled false
       proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+      //      applicationIdSuffix ".debug"
```
remove commented code

Trinkes commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/campaign/CampaignMapper.java:
```diff
@@ -0,0 +1,59 @@
+
+    if (found) {
+      String val = matcher.group(1);
+      //TODO should remove quotes and commas on the regex itself!
```
todo

brunovarelaAptoide committed:
Changes on Code Review

-Removed the comented code
- Removed a not used class

JoaoRaimundo commented on appcoins-ads/build.gradle:
```diff
@@ -46,12 +52,8 @@ dependencies {
-  api lib.reactivenetwork_rx2
-
-  api project(path: ':appcoins-contract-proxy')
+  api project(path: ':appcoins_net')
```
Can we improve this name, since the module is generic, does it needs to be appcoins related? And "net" part, can it be more explicit.

JoaoRaimundo commented on appcoins_net/src/main/java/com/appcoins/net/AppCoinsClient.java:
```diff
@@ -0,0 +1,47 @@
+package com.appcoins.net;
+
+public class AppCoinsClient implements AppCoinsConnection {
```
If this class has such relation with the ads model, does it make sense to in a different module? Its purpose doesn't exist out of the ads context.

JoaoRaimundo commented on appcoins_net/src/main/java/com/appcoins/net/AppCoinsConnection.java:
```diff
@@ -0,0 +1,8 @@
+package com.appcoins.net;
+
+interface AppCoinsConnection {
```
If this class has such relation with the ads model, does it make sense to in a different module? Its purpose doesn't exist out of the ads context.

JoaoRaimundo commented on appcoins_net/src/main/java/com/appcoins/net/AppCoinsHTTPClient.java:
```diff
@@ -0,0 +1,78 @@
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class AppCoinsHTTPClient implements Runnable {
```
Is this HttpClient AppCoins related to have its name dependent of it?

JoaoRaimundo commented on appcoins_net/src/main/java/com/appcoins/net/AppCoinsPingClient.java:
```diff
@@ -0,0 +1,50 @@
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class AppCoinsPingClient extends AppCoinsHTTPClient implements Runnable {
```
Is this PingClient AppCoins related to have its name dependent of it?

JoaoRaimundo commented on appcoins_net/src/main/java/com/appcoins/net/GetCampaignOperation.java:
```diff
@@ -0,0 +1,49 @@
+  public static final String TYPE = "type";
+
+  @Override
+  public String mapParams(String packageName, String versionCode, QueryParams queryParams) {
```
Can't we use the URI (java.net.URI) to do this mapping for us?

JoaoRaimundo commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -264,21 +226,27 @@ private void postponeSendProof() {
+            processCampaign(campaign);
+          });
+        } else {
+          Log.d("Message:", "No Connectivity Available: Delayed.");
```
Shoul we have an only for the purpose of logging something?

JoaoRaimundo commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -264,21 +226,27 @@ private void postponeSendProof() {
+        AppCoinsClientResponsePing pingResponse =
+            (AppCoinsClientResponsePing) appcoinsClientResponsePing;
+
+        if (pingResponse.HasConnection()) {
```
What happens if we don't have network? Does the ping mechanism makes any retries? Or the process is stopped until the application is closed and opened again?

JoaoRaimundo commented on toolbox/src/main/AndroidManifest.xml:
```diff
@@ -1,7 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.asf.appcoins.toolbox">
-
+  <uses-permission android:name="android.permission.INTERNET" />
```
Do we need this permission here when it's set up already in the modules imported that have the permission set up?

brunovarelaAptoide committed:
Appcoins_net merged

Merged appcoins_net with appcoins ads dev

mikieribeiro committed:
Merge Appcoins net with ads

-Removed <uses-permission android:name="android.permission.INTERNET"

-Removed”appcoins” from appcoins_net module

-Refractored AppCoinsPingClient to PingClient on appcoins_net/src/main/java/com/appcoins/net/AppCoinsPingClient.java

-Refractored AppCoinsHTTPClient to HTTPClient
 on appcoins_net/src/main/java/com/appcoins/net/AppCoinsHTTPClient.java

brunovarelaAptoide committed:
Remove Lamdas

Remove all Lambdas in project;
Java version downgrade to java 7
add a mechanism for checking connectivity retry

brunovarelaAptoide committed:
WalletUtils changes

changes

brunovarelaAptoide committed:
Changes Wallet Utils2

- changed appContext to activity

mikieribeiro committed:
Appcoins Merge

-CheckPreferences updated

brunovarelaAptoide committed:
changes runnable

brunovarelaAptoide committed:
Code Review changes

- Deleted interfaces
- Java 8 -> Java 7
- Changed Names
- Used URI to create query URL

brunovarelaAptoide committed:
added the interceptor e log class

- Added the interceptor and log class.

brunovarelaAptoide committed:
Merge pull request #16 from Aptoide/Merge-Appcoins-NET-With-Appcoins-ADS

Merge appcoins net with appcoins ads

brunovarelaAptoide committed:
Patch

- Applied patch.

brunovarelaAptoide committed:
Merge branch 'Merge-Appcoins-NET-With-Appcoins-ADS' into APPC-935-Remove-dependencies-from-other-SDK-modules

brunovarelaAptoide committed:
Changes

- Change package net to network
- Remove appcoins_net from publish.sh and settigs.gradle

JoaoRaimundo commented on appcoins-ads/build.gradle:
```diff
@@ -25,19 +27,25 @@ android {
   compileOptions {
-    targetCompatibility TARGET_COMPATIBILITY
-    sourceCompatibility SOURCE_COMPATIBILITY
+    targetCompatibility TARGET_COMPATIBILITY_ADS
```
Since we are no longer using lambdas, this can be removed

brunovarelaAptoide replied:
Removed

JoaoRaimundo commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/network/clients/GetCampaignHTTPClient.java:
```diff
@@ -0,0 +1,106 @@
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class GetCampaignHTTPClient implements Runnable {
```
Is this class an HTTPClient?

brunovarelaAptoide replied:
Changed for GetCampaignClient.

JoaoRaimundo commented on appcoins-net/src/main/AndroidManifest.xml:
```diff
@@ -0,0 +1,6 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```
Is this file needed?

brunovarelaAptoide replied:
Removed

JoaoRaimundo commented on gradle/android.gradle:
```diff
@@ -5,5 +5,8 @@ ext {
   SOURCE_COMPATIBILITY = 1.8
+
+  TARGET_COMPATIBILITY_ADS = 1.7
+  SOURCE_COMPATIBILITY_ADS = 1.7
```
Is this needed?

brunovarelaAptoide replied:
Removed

JoaoRaimundo commented on toolbox/build.gradle:
```diff
@@ -1,42 +1,126 @@
-    minSdkVersion MIN_SDK_VERSION
-    targetSdkVersion TARGET_SDK_VERSION
+  defaultConfig {
+    applicationId "com.appcoins.trivialdrivesample"
```
Please revert this to the original package name

JoaoRaimundo commented on toolbox/build.gradle:
```diff
@@ -1,42 +1,126 @@
+  defaultConfig {
+    applicationId "com.appcoins.trivialdrivesample"
+    minSdkVersion 21
+    targetSdkVersion 28
```
Please use the variables as it was defied before.

JoaoRaimundo commented on toolbox/build.gradle:
```diff
@@ -1,42 +1,126 @@
+      buildConfigField "String", "IAB_BIND_PACKAGE",
+          "\"" + project.GOOGLE_IAB_BIND_PACKAGE + "\""
+      buildConfigField "String", "IAB_UPDATE_ACTION",
+          "\"" + project.GOOGLE_IAB_UPDATE_ACTION+ "\""
```
Is this used on this demo app?

JoaoRaimundo commented on toolbox/build.gradle:
```diff
@@ -1,42 +1,126 @@
-  api project(':appcoins')
+  api 'com.android.support:appcompat-v7:28.0.0'
+  api 'com.google.code.gson:gson:2.8.2'
+  api "com.google.code.gson:gson:2.8.2"
```
is this used in here?

JoaoRaimundo commented on toolbox/src/main/java/com/asf/appcoins/toolbox/App.java:
```diff
@@ -2,16 +2,23 @@
-      AdvertisementSdkSingleton.getAdsSdk()
-          .init(this);
+
+      adsSdk = AdvertisementSdkSingleton.getAdsSdk();
```
Is this change needed?

JoaoRaimundo commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/network/clients/GetCampaignHTTPClient.java:
```diff
@@ -0,0 +1,106 @@
+    Uri campaignUri = Uri.parse(serviceUrl+"/campaign/listall?")
+        .buildUpon()
+        .appendQueryParameter(PACKAGE_NAME, packageName)
+        .appendQueryParameter(VER_CODE, versionCode + "")
```
Why don't we use String.valueOf() instead of concatenation?

brunovarelaAptoide committed:
Code Review Changes

-Changes for all the steps of the code review

JoaoRaimundo commented on appcoins_net/src/main/java/com/appcoins/net/GetCampaignOperation.java:
```diff
@@ -0,0 +1,49 @@
+  @Override
+  public String mapParams(String packageName, String versionCode, QueryParams queryParams) {
+    return "/campaign/listall?"
+        + PACKAGE_NAME
```
Can't we use the URI (java.net.URI) to do this mapping for us?

brunovarelaAptoide committed:
Code review - Key Removal

- Changes in properties
- Key removal from settings.gradle

JoaoRaimundo commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -48,29 +37,29 @@
   /** The runnnable taks that will be trigger periodically */
   private Runnable sendProof;
-  /** The handle to keep the runnable tasks that we be running within a certain period */
+  /** The handleRetryConnection to keep the runnable tasks that we be running within a certain period */
```
Find replace of string changed the javadoc. Please revert the change.

JoaoRaimundo commented on appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java:
```diff
@@ -48,29 +37,29 @@
   /** boolean indicating if we are already processing a PoA */
   private boolean processing;
-  /** The handle to keep the runnable tasks that we be running within a certain period */
+  /** The handleRetryConnection to keep the runnable tasks that we be running within a certain period */
```
Find replace of string changed the javadoc. Please revert the change.

brunovarelaAptoide committed:
Code Review Changes

- Changed the javadoc change to the original text.
